pkg/wallet/wallet.go

```diff
@@ -56,15 +57,15 @@ func NewWallet(location string) (*Wallet, error) {
 func NewWalletFromFile(path string) (*Wallet, error) {
 	file, err := os.Open(path)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to open wallet: %w", err)
```

Comment:
`s/failed to//` ? Do we have `path` here? Can it be bad (requiring `%q`)?

Reply:
Fixed.

Reply:
I see path is coming from https://pkg.go.dev/io/fs#PathError.Error in the resulting error string.